source/Octopus.Tentacle.Client/Scripts/ScriptServiceV3AlphaOrchestrator.cs

```diff
@@ -46,7 +45,27 @@ class ScriptServiceV3AlphaOrchestrator : ObservingScriptOrchestrator<StartScript
             this.logger = logger;
         }
 
-        protected override StartScriptCommandV3Alpha Map(StartScriptCommandV3Alpha command) => command;
+        protected override StartScriptCommandV3Alpha Map(ExecuteScriptCommand command)
+        {
+            IScriptExecutionContext executionContext = command switch
+            {
+                ExecuteKubernetesScriptCommand kubernetesScriptCommand => new KubernetesAgentScriptExecutionContext(kubernetesScriptCommand.Image, kubernetesScriptCommand.FeedUrl, kubernetesScriptCommand.FeedUsername, kubernetesScriptCommand.FeedPassword),
+                _ => new LocalShellScriptExecutionContext()
+            };
```

Comment:
The current V3Alpha as the `ScriptExecutionContext` to distinguish, so we look at the command type to create the relevant context